teuthology/orchestra/remote.py

```diff
@@ -26,7 +26,275 @@
 log = logging.getLogger(__name__)
 
 
-class Remote(object):
+class LocalOps(object):
```

Comment:
Maybe _Common_ is more suitable name for this kind of orchestration operation... 
Also, since you want to isolate this API maybe moving to separate module (remote) for better abstraction?

Reply:
A separate module makes sense to me. I also agree the name could be improved. Let's call it `RemoteOrchestration`?

Reply:
I've been thinking about a better name too. `Common` feels too generic and `RemoteOrchestration` doesn't specify why methods like `put_file()` and `get_file()` are in Remote and not here. Perhaps... `NonTransferOps`? I am not sure myself.

Reply:
Renamed.

Reply:
It is a bit strange to have RemoteOrchestration and overkill, it is in the module, orchestra.remote already.